project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject democracyworks/squishy "3.0.3-SNAPSHOT"
+(defproject democracyworks/squishy "3.1.0-SNAPSHOT"
```

Comment:
I think the AWS versions could probably be updated, maybe some of the other deps as well.

Reply:
Good call, I did forget to run `lein ancient`!